examples/colab/tf2_object_detection.ipynb

```diff
@@ -606,13 +615,15 @@
       "name": "Object Detection Inference on TF 2 and TF Hub",
       "private_outputs": true,
       "provenance": [],
-      "toc_visible": true
+      "toc_visible": true,
+      "include_colab_link": true
```

Comment:
Please also remove these additions

Reply:
I'll, and created a pull request, thank you!